### PR TITLE
Fix AXR Playback mode inaccessible

### PR DESCRIPTION
**Pull request by retiutut**
The issue is with BatteryInfo in the Aux widget when trying to playback CSV file (Cyton, Ganglion, or AXR). Was throwing this error:

```
java.lang.ClassCastException: OpenBCI_GUI$DataSourcePlayback cannot be cast to OpenBCI_GUI$BatteryInfoCapableBoard
        at OpenBCI_GUI$W_NovaAux.<init>(OpenBCI_GUI.java:21828)
```

Here is an initial fix. Maybe there is a better way... Open to suggestions.

This PR also addresses an issue where BrainFlow throws an error when trying to connect to board, but GUI shows "success" at the bottom. Now, it does a hard `return` and exits session init process and returns UI back to control panel.

### CODE REVIEW

File: OpenBCI_GUI/DataSourcePlayback.pde [hunk 1]
```diff
@@ -355,6 +356,19 @@ class DataSourcePlayback implements DataSource, AccelerometerCapableBoard, Analo
         return new int[0];
     }
 
+    @Override
+    public Integer getBatteryChannel() {
+        if (batteryChannelCache == null && underlyingBoard instanceof BatteryInfoCapableBoard) {
+            try {
+                batteryChannelCache = BoardShim.get_battery_channel(BoardIds.AURAXR_BOARD.get_code());
```
Comment (Andrey1994) [inline OpenBCI_GUI/DataSourcePlayback.pde:363]: I think you need to use board id of underlying board instead hardcoded auraxr

File: OpenBCI_GUI/OpenBCI_GUI.pde [hunk 2]
```diff
@@ -537,31 +537,15 @@ void initSystem() {
...
-        outputError("Failed to connect to data source fail point 1");
+        outputError("Failed to initialize board. Please check that the board is on and has power.");
+        controlPanel.open();
+        return;
     } else {
-        //initilize the GUI
+        //initilize the secondary topnav and all applicable widgets
         topNav.initSecondaryNav();
-
         wm = new WidgetManager(this);
```
Comment (Andrey1994) [inline OpenBCI_GUI/OpenBCI_GUI.pde:546]: Why is it removed? And what/where is it?

**Review by Andrey1994 — APPROVED** [review]

File: OpenBCI_GUI/OpenBCI_GUI.pde (hunk 2 above)
Comment (retiutut) [reply OpenBCI_GUI/OpenBCI_GUI.pde:546]: This part of session init code has extra code that did nothing. It has been consolidated in place.

File: OpenBCI_GUI/DataSourcePlayback.pde (hunk 1 above)
Comment (retiutut) [reply OpenBCI_GUI/DataSourcePlayback.pde:363]: I had issues accessing the method.

Comment (retiutut) [reply OpenBCI_GUI/DataSourcePlayback.pde]: Fixed. Had to cast to BoardBrainFlow

**Pull request merged**